SConstruct

```diff
@@ -112,6 +112,25 @@ def check_openssl():
             raise Exception("Didn't find any '%s' line in '$ %s'" %
                             (CHECK_LINE, CHECK_COMMAND))
 
+def setImplicitCache():
+    '''Use implicit_cache on some targets to improve build times
```

Comment:
This isn't even a quibble, but just a note on my workflow.

I use `'` entirely for string constants.  For comments, I deliberately use `"""` so my search is easier...

Looking at the libraries I have installed on this machine, this is a common choice, so I must be copying someone here.